dev-libs/libsecp256k1/libsecp256k1-0.2.0-r2.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools multilib-minimal
+
+MyPN=secp256k1
+DESCRIPTION="Optimized C library for EC operations on curve secp256k1"
+HOMEPAGE="https://github.com/bitcoin-core/secp256k1"
+SRC_URI="
+	https://github.com/bitcoin-core/secp256k1/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/bitcoin-core/secp256k1/commit/772e747bd9104d80fe531bed61f23f75342d7d63.patch -> ${PN}-PR1159-772e74.patch
```

Comment:
Given the PR is closed (https://github.com/bitcoin-core/secp256k1/pull/1159), I'd rather just pull the patch into `files/` so it's easier to inspect (and rebase in future as required).

Reply:
I generally try to avoid bloating the ebuild repository with assets that are fetchable on demand, but okay, I'll do as you ask.

Reply:
I don't think there's an obligation for Github to keep around orphaned commits forever either, fwiw.

Reply:
It's not orphaned; it's reachable from PR1159. In order to become orphaned, the pull request would have to be deleted, and I don't believe GitHub ordinarily deletes pull requests. But no matter; I've already plopped a patch file into `${FILESDIR}`.

Reply:
You're right -- https://stackoverflow.com/questions/15261880/does-github-garbage-collect-dangling-commits-referenced-in-pull-requests.